src/site/paragraphs/q_a_section.drupal.liquid

```diff
@@ -21,6 +21,6 @@
     {% include "src/site/paragraphs/q_a.collapsible_panel.drupal.liquid" questions = entity.fieldQuestions %}
 {% else %}
     {% for question in entity.fieldQuestions %}
-      {% include "src/site/paragraphs/q_a.drupal.liquid" paragraph = question.entity sectionHeader = entity.fieldSectionHeader %}
+      {% include "src/site/paragraphs/q_a.drupal.liquid" entity = question.entity sectionHeader = entity.fieldSectionHeader %}
```

Comment:
i needed to change this to make the templates consistent with `entity` instead of `paragraph`